Arduino/McLighting/McLighting.ino

```diff
@@ -692,6 +771,93 @@ void setup() {
     getStatusJSON();
   });
 
+  server.on("/pixels", []() {
+
+    bool updateStrip = false;
+    if(server.hasArg("ct")){
+      uint16_t pixelCt = server.arg("ct").toInt();
+      if (pixelCt > 0) {
+        WS2812FXStripSettings.stripSize = pixelCt;
+        updateStrip = true;
+        DBG_OUTPUT_PORT.printf("/pixels: Count# %d\n", pixelCt);
+      }
+    }
+    if(server.hasArg("rgbo")){
+      String RGBOrder = server.arg("rgbo");
+      DBG_OUTPUT_PORT.print("/pixels: RGB Order# ");
+      if (RGBOrder == "grb") {
```

Comment:
I'm not an programming expert in Arduino/c/c++, but couldn't this be better solved via an extra `if` on the very bottom of this function, where we set
```
updateStrip = true;
DBG_OUTPUT_PORT.println(RGBOrder);
```
if  this if-else-ladder did not fail?
Would make this a little bit more cleaner.
But as I said: Not an expert.

Reply:
I'm still learning, no one is an expert. What I found is new webserver code (core 2.5.2) sends each argument separately, that's why I used a Boolean as a quick check. We also do validation of each input received.

Reply:
Having one bool do the job is better than multiple if statement checks. **updateStrip** does valid input check for multiple inputs, so acts as an "OR" for now.